node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -1,5 +1,7 @@
 package coop.rchain.node.configuration
 
+import cats.syntax.all._
```

Comment:
Is this import now used?

Reply:
> Is this import now used?

Yes, for `asRight` syntax method
https://github.com/rchain/rchain/blob/5d7d53ff6b38ec3d093d3d3410a3775a0f488db6/node/src/main/scala/coop/rchain/node/configuration/Configuration.scala#L87